frontend/src/app/components/containers/inspections/edit/inspection-edit.tsx

```diff
@@ -193,10 +193,12 @@ const InspectionEdit: FC = () => {
   const confirmCancelChanges = useCallback(() => {
     form.reset();
 
-    if (isEditMode && id) {
+    if (id) {
```

Comment:
Your conditionals had a bit of redundant logic, isEditMode is irrelevant here:

If (isEditMode && id)
  navigate(`/inspection/${id}`);
else if (id)
  navigate(`/inspection/${id}`);
  
  Updated on your behalf since you're away this morning and we wanted to get these into test :)